open-metadata-distribution/open-metadata-assemblies/src/main/assemblies/omag-server.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!-- SPDX-License-Identifier: Apache-2.0 -->
+
+<assembly xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+          xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 http://maven.apache.org/xsd/assembly-1.1.2.xsd">
+    <formats>
+        <format>tar.gz</format>
+        <format>dir</format>
+    </formats>
+    <id>omag-server</id>
+    <baseDirectory>omag-server-${open-metadata.version}</baseDirectory>
+    <fileSets>
+        <fileSet>
+            <directory>../omag-server/target</directory>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>*.jar</include>
+                <include>*.sh</include>
+
+            </includes>
+            <fileMode>0755</fileMode>
+            <directoryMode>0755</directoryMode>
+        </fileSet>
+        <fileSet>
+            <directory>../omag-server</directory>
+            <outputDirectory>.</outputDirectory>
+            <includes>
+                <include>*.md</include>
+
+            </includes>
+            <fileMode>0755</fileMode>
+            <directoryMode>0755</directoryMode>
+
+        </fileSet>
+
+    </fileSets>
+</assembly>
```

Comment:
I keep getting those nulls at end of line too. They show up as red on a review. I think there's a setting in intellij to fix. I've NOT tested but there is Preferences->Editor->General->Ensure line feed at file end on Save ? Maybe...